data/templates/markdown/elements/table-of-contents.md.twig

```diff
@@ -1,10 +1,18 @@
 {% block table_of_contents %}
 {% set url_base = (url_base) ? url_base : '../' %}
+{% set non_empty_namespaces = [] %}
 {% if node.children|length > 0 %}
+{% for namespace in node.children|sort_asc %}
+{% if ( namespace.children|length > 0 ) or ( namespace.traits|length > 0 ) or ( namespace.interfaces|length > 0 ) or ( namespace.classes|length > 0 ) or ( namespace.constants|length > 0 ) or ( namespace.functions|length > 0 ) %}
```

Comment:
I believe for `constants`, `properties` and `methods`.. we need to use the methods `constants(namespace)`, `properties(namespace)` and `methods(namespace)` respectively.

`namespace.methods` would not include inherited methods for example, while `methods(namespace)` would.